components/sprayproxy/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 resources:
-  - https://github.com/redhat-appstudio/sprayproxy/config?ref=076942559b739729c156b3d64f87efda1731ab90
+  - https://github.com/redhat-appstudio/sprayproxy/config?ref=0eece5fdaf5c84ed79c852b7c9c7118ced5d4390
 
 namespace: sprayproxy
 
 images:
   - name: ko://github.com/redhat-appstudio/sprayproxy
     newName: quay.io/redhat-appstudio/sprayproxy
-    newTag: 076942559b739729c156b3d64f87efda1731ab90
+    newTag: 0eece5fdaf5c84ed79c852b7c9c7118ced5d4390
```

Comment:
fyi this is available now per https://quay.io/repository/redhat-appstudio/sprayproxy?tab=tags

Reply:
there is for the main branch @avinal but it does not work as I understand it for the multi cluster branch

Reply:
and per the open PRs at https://github.com/redhat-appstudio/infra-deployments/pulls you'll see there is no PRs pending to update the sprayproxy's sha ref

Reply:
Got it. I was not aware of the multi-cluster branch.